askama_shared/src/lib.rs

```diff
@@ -216,6 +223,7 @@ struct General<'a> {
     #[cfg_attr(feature = "serde", serde(borrow))]
     dirs: Option<Vec<&'a str>>,
     default_syntax: Option<&'a str>,
+    generated: Option<bool>,
```

Comment:
```suggestion
    #[serde(default)]
    generated: bool,
```

Any particular reason for the `Option`? :)